engine/graphics/src/dmsdk/graphics/graphics.h

```diff
@@ -396,6 +397,7 @@ namespace dmGraphics
         TYPE_FLOAT_VEC3       = 13,
         TYPE_FLOAT_MAT2       = 14,
         TYPE_FLOAT_MAT3       = 15,
+        TYPE_IMAGE_2D         = 16,
```

Comment:
An "image" doesn't have a sampler and is one of the most common ways of passing data into a compute shader or as a data storage for writing.